webpack/assets/javascripts/react_app/components/AuditsList/__tests__/__snapshots__/ExpansiveView.test.js.snap

```diff
@@ -7,6 +7,7 @@ exports[`ExpansiveView rendering render audit record changes 1`] = `
   <DiffContainer
     newText="<h1>Hello World..</h1>"
     oldText="<h1>Hello..</h1>"
+    patch=""
```

Comment:
why?

Reply:
beacuse the diffView now supports both old,new Text and patch inputs. only one of the options is used at a time